bluetooth/src/commonMain/kotlin/device/AdvertisementData.kt

```diff
@@ -30,3 +31,13 @@ interface BaseAdvertisementData {
 }
 
 expect class AdvertisementData : BaseAdvertisementData
+
+val BaseAdvertisementData.description: String get() = listOfNotNull(
+    name?.let { "Name: $it" },
+    manufacturerId?.let { "ManufacturerId: $it" },
+    manufacturerData?.let { "ManufacturerData: $it" },
+    if (serviceUUIDs.isEmpty()) null else { "ServiceUUIDS: ${serviceUUIDs.joinToString(", ") { it.uuidString }}" },
```

Comment:
I've extract into helper methods for List<UUID> as code inside `${}` hard to read/review/edit